types/ion-rangeslider/ion-rangeslider-tests.ts

```diff
@@ -0,0 +1 @@
+import "ion-rangeslider";
```

Comment:
The previous version had a test file with significantly more content. Please add or restore the tests.